### DISCUSSION TITLE
After success build and publish I don't find build in app store developer

### DISCUSSION THREAD

**Discussion by connect-linkss** [codemagic-ci-cd/codemagic-docs#2816] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

- **first** I develop flutter app and deploy in play store
- I faced problem when build app in codemajic becouse select xcode 16
- when select xcode 15.2 the build success and publishing success
- but when go to review change I dont see any build in dashboard

### Expected results

I just want to know why this is happening and get some tips to solve the problem.

### Actual results

- I faced problem when build app in codemajic becouse select xcode 16
- when select xcode 15.2 the build success and publishing success
- but when go to review change I dont see any build in dashboard

### Build id (optional)

672b53f7d43867d962c4d881

**Comment by dtrdic** [comment]
Hi @connect-linkss i see the binary was successfully uploaded to App Store Connect for processing but if the corresponding build is not available in App Store Connect that usually means that the uploaded application has one or more issues. Please check your email for direct feedback from Apple

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
